frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -21,9 +21,21 @@ import {
 } from '@patternfly/react-core';
 
 import { history } from './router';
+import { Dropdown } from '../utils';
 
-const CATEGORY_URL_PARAM = 'category';
-const KEYWORD_URL_PARAM = 'keyword';
+export const FilterTypes = {
+  category: 'category',
+  keyword: 'keyword',
+};
+
+export const DisplayTypes = {
+  groupBy: 'groupBy',
+};
+
+export const groupByTypes = {
+  Operator: 'Operator',
```

Comment:
I just realized that `tile-view-page` is a generic component. You must parameterize the grouping strategy using props and control it from the parent.